doc/CONFIGURE.md

```diff
@@ -0,0 +1,205 @@
+    you may want them to only run on a specially configured machine, while preventing all other jobs from using that machine.
+    To do so, you would restrict where the test jobs may run by giving them a label expression matching that machine. 
+    Furthermore, if you set the # of executors value to 1, you can ensure that only one performance test will execute at 
+    any given time on that machine; no other builds will interfere.
+* **Launch method:** Select *Launch agent agents via SSH* Starts a agent by sending commands over a secure SSH connection.
+
+![](images/ssh-node-basic-config.png)
+
+## Configure Launch agent agents via SSH
+
+Once you selected the **Launch method** to **Launch agent agents via SSH**, you can configure your SSH agent settings.
+
+### Required settings
+
+* **Host:** Hostname or IP of the agent, it should be resolved and reachable from the Jenkins instance. 
+* **Credentials:** Select the credentials to be used for logging in to the remote host. See [Integration with SSH Credentials Plugin](#integration-with-ssh-credentials-plugin)
+* **Host Key Verification Strategy:** Controls how Jenkins verifies the SSH key presented by the remote host whilst connecting. See [Host Key Verification Strategy](#host-key-verification-strategy)
+
+![](images/ssh-node-launcher-config.png)
+
+### Advanced settings
+
+* **Port:** The TCP port on which the slave's SSH daemon is listening, usually 22.
+* **JavaPath** This java Path will be used to start the jvm. (/mycustomjdkpath/bin/java ) If empty Jenkins will search java command in the agent.
+* **JVM Options** Additional arguments for the JVM, such as -Xmx or GC options.
+* **Prefix Start Agent Command** What you enter here will be prepended to the launch command.
+* **Suffix Start Agent Command** What you enter here will be appended to the launch command.
+* **Connection Timeout in Seconds** Set the timeout value for ssh agent launch in seconds.
+If empty, it will be reset to default value. This will only set the timeout for agent launching; once launched,
+the timeout will not apply.
+* **Maximum Number of Retries** Set the number of times the SSH connection will be retried if the initial connection results in an error.
+If empty, it will be reset to default value.
+* **Seconds To Wait Between Retries** Set the number of seconds to wait between retry attempts of the initial SSH connection.
+* **Use TCP_NODELAY flag on the SSH connection** Enable/Disables the TCP_NODELAY flag on the SSH connection.
+If set, disable the Nagle algorithm. This means that segments are always sent as soon as possible,
+even if there is only a small amount of data. When not set, data is buffered until there is a sufficient amount to send out,
+thereby avoiding the frequent sending of small packets, which results in poor utilization of the network.
+* **Remoting Work directory** The Remoting work directory is an internal data storage, which may be used by Remoting to store caches,
```

Comment:
I don't think "work" should be capitalized here, or you should make the use of capitals consistent in this list.